<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Agregar</class>
 <widget class="QDialog" name="Agregar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 127);</string>
  </property>
  <widget class="QLineEdit" name="lineEditNombre">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCantidad">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHorario">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Garamond&quot;;
</string>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Garamond&quot;;
</string>
   </property>
   <property name="text">
    <string>Cantidad</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Garamond&quot;;
</string>
   </property>
   <property name="text">
    <string>Horario</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonVolver">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Garamond&quot;;</string>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAgregar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Garamond&quot;;</string>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
